src/aleph/http.clj

```diff
@@ -336,6 +337,9 @@
      by [clj-http](https://github.com/dakrone/clj-http), and returns a deferred representing
      the HTTP response.  Also allows for a custom `pool` or `middleware` to be defined.
 
+     Putting the returned deferred into an error state will cancel the underlying request if it is
+     still in flight.
+
```

Comment:
Maybe add a note to the effect that partial processing may still have occurred.

Reply:
Good idea! Did that and also touched up the explanation to make explicit that cancellation means closing of the TCP connection. It now also mentions that cancellation will have no effect if the request had already completed by the time of invoking `cancel-request!`.